dpsprep/outline.py

```diff
@@ -1,23 +1,33 @@
 from loguru import logger
 from pdfrw import PdfName, PdfDict, IndirectPdfDict
 import djvu.sexpr
+import re
 
 from .sexpr import SExpressionVisitor
 
 
 # Based on
 # https://github.com/pmaupin/pdfrw/issues/52#issuecomment-271190546
 class OutlineTransformVisitor(SExpressionVisitor):
+    def __init__(self, toc_pg_offset : int, total_pages : int):
+        self.toc_pg_offset = toc_pg_offset
+        self.total_pages = total_pages
+        super().__init__()
     def visit_plain_list(self, node: djvu.sexpr.StringExpression, parent: IndirectPdfDict):
         title, page, *rest = node
         # I have experimentally determined that we need to translate page indices. -- Ianis, 2023-05-03
```

Comment:
This comment refers to the approach that you are trying to replace. It should be either deleted or updated.

Reply:
Updated.